tests/test_util/aiohttp.py

```diff
@@ -57,7 +57,7 @@ def call_count(self):
         return len(self.mock_calls)
 
     @asyncio.coroutine
-    def match_request(self, method, url, *, auth=None):
+    def match_request(self, method, url, *, auth=None, **kwargs):
```

Comment:
I don't want to use `kwargs` if we know exactly what keywords to expect. By adding uncertainty we add tech debt.